cpp/src/io/orc/writer_impl.cu

```diff
@@ -737,6 +737,7 @@ std::vector<std::vector<uint8_t>> writer::impl::gather_statistic_blobs(
       default: desc->stats_dtype = dtype_none; break;
     }
     desc->num_rows         = columns[i].data_count();
+    desc->num_values       = columns[i].data_count();
```

Comment:
`num_rows` and `num_values` both are getting assigned from `columns[i].data_count();`, how this would hold in case of nested types

Reply:
The actual number of values is different from number of rows for list columns. But it is the same in case of non-list columns. In parquet writer the distinction has been made at the time of adding list column support and the values are set differently. In case of ORC, currently they are the same. Line 739 would need to be updated when list column support is added.